docs/roadmap.md

```diff
@@ -27,22 +29,22 @@ tenants:
 
 We have all the metrics to track how many series, samples and queries each tenant is sending but don't have dashboards that help with this. We plan to have dashboards and UIs that will help operators monitor and control each tenants usage out of the box.
 
-## Downsampling and Per tenant/metric retention
+## Downsampling
+Downsampling means storing fewer samples, e.g. one per minute instead of one every 15 seconds.
+This makes queries over long periods more efficient. It can reduce storage space slightly if the full-detail data is discarded.
 
-Currently, we only support a single retention period for all metrics and tenants. For most operators, the ability to set per tenant retention and also custom retention for subsets of metrics is important. We will add support per tenant and metric retention policies. Also, we currently store all the samples we ingested, and there is no way to reduce the resolution for the metrics. We plan to add downsampling to allow users to store less data when needed.
+## Per-metric retention
 
-## Soft Multitenancy
+Cortex blocks storage supports deleting all data for a tenant after a time period (e.g. 3 months, 1 year), but we would also like to have custom retention for subsets of metrics (e.g. delete server metrics but retain business metrics).
 
-Currently our multitenancy allows a tenant to view _all_ their metrics and only their metrics. There is no way for an "admin" tenant to view all the metrics in the system but for particular teams to only view theirs. This is another feature we plan to add into Cortex.
-
-## Exemplar and Prometheus metadata support
```

Comment:
> also add persistence of the metadata (right now it's only in-mem)

This has not been done, but not sure if we want it. At least, doesn't look a priority until Prometheus will keep metadata only in-memory as well.

Reply:
Good point.  For consistency, metadata should match the point in history where you are looking, but it doesn't strike me as a hi-priority feature.